docs/okeps/okep-5861-two-ovnks.md

```
@@ -0,0 +1,99 @@
+If unset, behavior and paths remain as today for backward compatibility.
+
+If set, all file paths, sockets, and PID files are derived from the instance ID.
+​
+
+OVS/OVN 
+
+Shared OVS on the DPU may serve both instances.
+
+The relevant bits for OVN inside the OVSDB are the following external_ids:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)